### ISSUE TITLE
Error using typing_extensions.Protocol in python3.6

### ISSUE THREAD

**Issue by jacobbogdanov** [microsoft/pyright#763]
**Describe the bug**
`pyright` vscode extension incorrectly reports `Use of "Protocol" requires Python 3.7 or newer`, when using python3.6 and importing `Protocol` from `typing_extensions`.

This seems to be related to #73.

**To Reproduce**
open the code below in the vscode extension.
       
**Expected behavior**
importing `Protocol` from `typing_extensions` should not trigger an error.

**Screenshots or Code**
Using the following code (in `test.py`) as an example
```python
from typing_extensions import Protocol

class Duck(Protocol):  # the vscode extension says 'Use of "Protocol" requires Python 3.7 or newer'
    def quack(self) -> str:
        ...

class MyDuck:
    def quack(self) -> str:
        return "quack quack"

def speak(duck: Duck) -> None:
    print(duck.quack())
```
`pyright test.py` reports
```
Found 1 source file
0 errors, 0 warnings 
Completed in 0.725sec
```

<img width="665" alt="Screen Shot 2020-06-25 at 12 05 43 PM" src="https://user-images.githubusercontent.com/48320751/85755787-3bc44400-b6dc-11ea-8ea1-c9e9f45e5937.png">

**VS Code extension or command-line**
pyright version 1.1.43
pyright vscode extension version v1.1.45

**Comment by erictraut** [comment]
Thanks for the bug report. This will be fixed in the next version of Pyright.

**Comment by erictraut** [comment]
This is now fixed in version 1.1.47, which I just published.
